src/cmds/pbsdsh.c

```diff
@@ -95,6 +107,96 @@ bailout(int sig)
  * @return - Void
  *
  */
+
+
+int
+host_match(char *line, char *host)
+{
+	if (NULL == host ) return 0;
```

Comment:
You call strlen() on line below.  Maybe check if neither argument is NULL?  Also, please remove the space before the ')'

I believe our coding standard has if blocks on the line under the if statement regardless if it is one line or more.

Reply:
Could add  NULL check here. 
Btw, i didnt find any standard related to simple 'if' block if there is a simple clause within it. indeed there is 'else if' standard. 
ref: https://openpbs.atlassian.net/wiki/spaces/DG/pages/1481670667/OpenPBS+Coding+Standards
you have the finally say, so i'll add a line break here.

Reply:
Sorry about that.  It's been a bit since I read the standard.  I could have sworn it was there.  I guess it was just my own personal preference.  Thanks for making the change.  I find it easier to read.